fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingServiceImpl.java

```diff
@@ -160,14 +152,15 @@ public GenericResultsetData retrieveGenericResultset(final String name, final St
             final boolean isSelfServiceUserReport) {
 
         final long startTime = System.currentTimeMillis();
-        LOG.info("STARTING REPORT: {}   Type: {}", name, type);
+        LOG.info("STARTING REPORT: {}   Type: {}", name.replaceAll("[\n\r\t]", "_"), type.replaceAll("[\n\r\t]", "_"));
```

Comment:
Let's do a log level check here to make sure the replacement is only executed if the logger is configured for info level. The typical pattern here is:
```
if (LOG.isInfoEnabled()) {
     ...
}
```
Also, I honestly think info is way too verbose here, let's decrease it to debug level.